### PR TITLE
Create COVERAGES.md

### PR DESCRIPTION
**Pull request by mamonu**

### CODE REVIEW

**Comment by mamonu** [comment]
done changes requested

File: COVERAGES.md [hunk 1]
```diff
@@ -1,15 +1,17 @@
```
Comment (athityakumar) [inline COVERAGES.md]:
```markdown
# COVERAGES
```

**Comment by mamonu** [comment]
added header and edited the first sentence to be more natural.

File: COVERAGES.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+A documentation of all the places on the World Wide Web, where `colorls` has been featured:
+
+
+- [YouTube : File Icons on Linux Terminal](https://m.youtube.com/watch?v=NESi45Q2mHg)
+- [GitHub : Awesome README](https://github.com/matiassingers/awesome-readme)
+- [HackerNews : Colorls – A Ruby script that beautifies the terminal's ls command](https://news.ycombinator.com/item?id=14717341)
+- [GitHub : A curated list of Terminal frameworks, plugins & resources for CLI lovers](https://github.com/k4m4/terminals-are-sexy)
+- [Facebook : colorls story](https://m.facebook.com/story.php?story_fbid=1972265916349994&id=1884586571784596)
+- [GitHub : nerd-fonts](https://openruby.com/pages/98288680-colorls-a-ruby-script-that-beautifies-the-terminals-ls)
```
Comment (athityakumar) [inline COVERAGES.md]: Seems like a wrong caption for the link.

File: COVERAGES.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+- [GitHub : nerd-fonts](https://openruby.com/pages/98079215-athityakumar-slash-colorls-a-ruby-script-that-beautifies-the)
```
Comment (athityakumar) [inline COVERAGES.md]: Seems like a wrong caption for the link.

**Comment by mamonu** [comment]
fixed the wrong links

**Pull request merged**